src/vcpkg/base/files.cpp

```diff
@@ -3812,6 +3824,39 @@ namespace vcpkg
 #endif // ^^^ !_WIN32
         }
 
+        int64_t file_time_now() const override
```

Comment:
Hmmm this doesn't seem to actually touch the filesystem and thus it could just be a nonmember?

Reply:
Could be. But if you want to implement a fake filesystem you maybe want to change its implementation.